docs/running-on-kubernetes.md

```diff
@@ -31,20 +42,27 @@ If you wish to use pre-built docker images, you may use the images published in
   <td>Spark Executor Image</td>
   <td><code>kubespark/spark-executor:v2.1.0-kubernetes-0.1.0-alpha.2</code></td>
 </tr>
+<tr>
+  <td>Spark Initialization Image</td>
+  <td><code>kubespark/spark-init:v2.1.0-kubernetes-0.1.0-alpha.2</code></td>
+</tr>
 </table>
 
-You may also build these docker images from sources, or customize them as required. Spark distributions include the Docker files for the driver and the executor at
-`dockerfiles/driver/Dockerfile` and `dockerfiles/executor/Dockerfile`, respectively. Use these Docker files to build the
-Docker images, and then tag them with the registry that the images should be sent to. Finally, push the images to the
-registry.
+You may also build these docker images from sources, or customize them as required. Spark distributions include the
```

Comment:
We can follow up in a separate commit for documentation on creating custom Docker images.